nixos/modules/services/monitoring/prometheus/alertmanager.nix

```diff
@@ -144,9 +156,14 @@ in {
       systemd.services.alertmanager = {
         wantedBy = [ "multi-user.target" ];
         after    = [ "network.target" ];
+        preStart = ''
+           ${lib.getBin pkgs.envsubst}/bin/envsubst -o /tmp/alert-manager-substituted.yaml" \
```

Comment:
We might want to set a umask here, and also below explicitly enable PrivateTmp instead of depending on the implied PrivateTemp.